### DISCUSSION TITLE
I have a Mac MINI m2 and an Awus 1900 ALFA

### DISCUSSION THREAD

**Discussion by fsocietynpc** [chris1111/Wireless-USB-Big-Sur-Adapter#280] [Q&A]
So i've basically followed all the tutorial, and disabled SIP. 

Downloaded D-Link Utility Package and create the folder with my Network's name and Password and at the end i have a succesful installation without any error. But nothing appears i can't seem to find the approval in the Security and Privacy of the Extension and nothing pops up on the top of my screen with the Wifi ICON. Installation is succesful but nothing seems to work.

Anyone can help me thx, My config is a Mac Mini M2 with Mac Os Ventura 13.2

**Comment by chris1111** [comment]
> So i've basically followed all the tutorial, and disabled SIP. 
> 
> 
> 
> Downloaded D-Link Utility Package and create the folder with my Network's name and Password and at the end i have a succesful installation without any error. But nothing appears i can't seem to find the approval in the Security and Privacy of the Extension and nothing pops up on the top of my screen with the Wifi ICON. Installation is succesful but nothing seems to work.
> 
> 
> 
> Anyone can help me thx, My config is a Mac Mini M2 with Mac Os Ventura 13.2

It doesn’t work on Apple silicone M1 and M2
Read the page about that.
